indicator_ip/indicator_ip_menu.py

```diff
@@ -60,6 +62,22 @@ def create_menu(self):
         sep.show()
         menu.append(sep)
 
+
+        # get all refresh timer
+        group = None
+        for refresh_time in ["[OFF] autorefresh", "15 sec", "1 min", "1 hour"]:
```

Comment:
Here you have created a section in the main dialog.
It would have been better to have a dropdown.

- options
  - auto refresh
    - disabled
    - 15 sec
    - 1 min
    - 1 hour
    - ... 
So in the futur we can add more options.